### DISCUSSION TITLE
Add the have:need fields to the "Buy List (Only tier x)" sections.

### DISCUSSION THREAD

**Discussion by VeloxAzo** [xanthics/gw2craft#53] [Ideas]
Add the have:need fields to the "Buy List (Only tier x)" sections. 

I found myself wanting this while trying out the normal guide for Huntsman crafting. It asked for an absurd amount of intermediary products to be purchased on the TP and my crafting level was already in the 100s so I didn't need some of them anyway. 

So I'm asking for 
![image](https://user-images.githubusercontent.com/11967899/139596823-58b4f670-7dff-4070-a648-64c476fafb46.png)

To look like 
![image](https://user-images.githubusercontent.com/11967899/139596808-b590efe5-b19d-4020-97f0-88af3fb44e42.png)

**Comment by xanthics** [comment]
I like the idea but the way the site is currently written makes it difficult/convoluted to cleanly add that feature.  As I would want it to update the input boxes at each place they appear.  I'll look further in to the changes this week and have a firmer answer by next week.

**Reply by xanthics** [reply]
This isn't something that I want to add without rewriting the generated pages in to a more modern template and design style.  For now, that is a lot more work than what I want to be doing on the site at this time.
